File: src/dotnet/Src/ButterflyNet.Parser/Scopes/DefinitionTermScope.cs
namespace ButterflyNet.Parser.Scopes {
	public class DefinitionTermScope : BlockScope {
		public override void Open(ButterflyAnalyzer analyzer) {
			analyzer.OpenDefinitionTerm();
		}

		public override void Close(ButterflyAnalyzer analyzer) {
			analyzer.CloseDefinitionTerm();
		}
	}
}

File: src/js/src/scopes/DefinitionTermScope.js
function DefinitionTermScope() {
	this.type = ScopeTypeCache.definitionTerm;
	this.level = ScopeLevel.block;
	this.canNestText = true;
	
	this.open = function(analyzer) {
		analyzer.openDefinitionTerm();
	};
	this.close = function(analyzer) {
		analyzer.closeDefinitionTerm();
	};
}

extend(Scope, DefinitionTermScope);